src/view/document.js

```diff
@@ -256,6 +256,17 @@ export default class Document {
 	}
 
 	/**
+	 * Focuses document. It will focus {@link engine.view.EditableElement EditableElement} that is currently having
+	 * selection inside.
+	 *
+	 * @fires engine.view.Document#focus
+	 */
+	focus() {
+		this.fire( 'focus' );
```

Comment:
This `focus` event has no attributes which `focus` event fired from the `FocusObserver` has. This is why why I called `focus()` in the focus manual  test I get an error from this line: https://github.com/ckeditor/ckeditor5-engine/blob/master/tests/view/manual/focusobserver.js#L18